### DISCUSSION TITLE
How to configure VSCode for opentelemetry-cpp developement / debugging

### DISCUSSION THREAD

**Discussion by mdb-gtenrreiro** [open-telemetry/opentelemetry-cpp#1968] [Q&A]
I am looking for information on how to configure VSCode to make it work with opentelemetry-cpp, so that I can get IntelliSense support for example ? or get the debugger to display variables. ? 
In my case for example, if I open an example file from the /examples folder, some of the includes at the top are not found, and I am not sure how to configure VSCode to figure out where the includes are ? 

UPDATE: Nevermind, it looks like I had Clangd as well as the C++ extension, and Clangd didn't understand where the includes were coming from.

**Comment by owent** [comment]
FYI:

```json
  "cmake.configureSettings": {
    "CMAKE_TOOLCHAIN_FILE": "D:/prebuilt/vcpkg/scripts/buildsystems/vcpkg.cmake",
    "VCPKG_TARGET_TRIPLET": "x64-windows-static-md",
    "VCPKG_CRT_LINKAGE": "dynamic",
    "WITH_OTLP": "ON",
    "WITH_OTLP_GRPC": "ON",
    "WITH_OTLP_HTTP": "ON",
    // "WITH_METRICS_PREVIEW": "ON",
    "WITH_LOGS_PREVIEW": "ON",
    "WITH_ELASTICSEARCH": "ON",
    "WITH_ASYNC_EXPORT_PREVIEW": "ON"
  },
  "cmake.configureOnEdit": false,
  "cmake.configureEnvironment": {
    "VCPKG_ROOT": "D:/prebuilt/vcpkg"
  }
}
```

It's my settings and I installed vcpkg in D:/prebuilt/vcpkg.
